src/lib/gadgets/common.ts

```diff
@@ -1,23 +1,43 @@
 import { Provable } from '../provable.js';
-import { Field } from '../field.js';
+import { Field, FieldConst } from '../field.js';
+import { TupleN } from '../util/types.js';
+import { Snarky } from '../../snarky.js';
+import { MlArray } from '../ml/base.js';
 
 const MAX_BITS = 64 as const;
```

Comment:
I am just seeing this - I think we should clarify what `MAX_BITS` is for (rotation gate, I believe?)

Reply:
yeah it probably doesn't belong in the `common.ts` file, complely out of context here

Reply:
will address in next PR